app/models/report.rb

```diff
@@ -303,8 +303,7 @@ def list_proquest_files(collection)
   def list_student_submitted_metadata(collection)
     result = []
     collection.order(:grad_date).uniq.each do |record|
-      next unless record.versions.present? && record.versions.first.whodunnit
-      next if record.versions.first.whodunnit == 'registrar'
+      next unless student_initiated_record?(record)
```

Comment:
This is fine for now - I wonder whether I'll end up refactoring it as part of ETD-415 if both reports need the same logic, but I don't think it makes sense to implement that way until multiple needs for it exist (just please don't be surprised if you see that in code review :-) )